test/index.js

```diff
@@ -70,7 +70,9 @@ describe('stylelint-webpack-plugin', function () {
 
     return pack(assign({}, baseConfig, config))
       .then(function (stats) {
-        expect(stats.compilation.errors).to.have.length(2);
+        expect(stats.compilation.errors).to.have.length(1);
+        expect(stats.compilation.errors[0]).to.contain('test/fixtures/test7/_second.scss');
+        expect(stats.compilation.errors[0]).to.contain('test/fixtures/test7/test.scss');
```

Comment:
Hm, that doesn't work, since that's not the full content of the string:

```
1) stylelint-webpack-plugin works with multiple source files:
     AssertionError: expected [ Array(1) ] to include 'test/fixtures/test7/_second.scss'
      at ~/projects/stylelint-webpack-plugin/test/index.js:74:45
```